app/domain/audits/save_audit.rb

```diff
@@ -0,0 +1,51 @@
+module Audits
+  class SaveAudit
+    def self.call(*args)
+      new(*args).call
+    end
+
+    attr_accessor :attributes, :audit, :content_item, :success, :user
+
+    def initialize(attributes:, content_id:, user_uid:)
+      self.attributes = attributes.to_h
```

Comment:
to_h needs to be done outside this method.

Reply:
Thanks Pablo - addressed here: https://github.com/alphagov/content-performance-manager/pull/346/commits/a4bf9015133ca999098d1c0c99c8ec281d483f01#diff-bcf8bfffc9defab9b861ec826b66da7aR9